src/Psalm/Output/Compact.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace Psalm\Output;
+
+use Psalm\Config;
+use Psalm\Output;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class Compact extends Output
+{
+    /**
+     * {{@inheritdoc}}
+     * @psalm-suppress PossiblyNullReference
+     */
+    public function create(): string
+    {
+        /** @var BufferedOutput|null $buffer */
+        $buffer = null;
+
+        /** @var Table|null $table */
+        $table = null;
+
+        /** @var string|null $current_file */
+        $current_file = null;
+
+        $output = [];
+        foreach ($this->issues_data as $i => $issue_data) {
+            if (!$this->show_info && $issue_data['severity'] === Config::REPORT_INFO) {
+                continue;
+            } elseif (is_null($current_file) || $current_file !== $issue_data['file_name']) {
+                // If we're processing a new file, then wrap up the last table and render it out.
+                if ($buffer !== null) {
+                    $table->render();
+                    $output[] = $buffer->fetch();
+                }
+
+                $output[] = 'FILE: ' . $issue_data['file_name'] . "\n";
+
+                $buffer = new BufferedOutput();
+                $table = new Table($buffer);
+                $table->setColumnMaxWidth(3, 70);
```

Comment:
Looks like this isn't available in symfony/console 3.x

Reply:
Available since 4.2

Reply:
Replaced with an alternate wrapping implementation.